playbooks/roles/nghttpx/files/snakeoil.pem

```diff
@@ -0,0 +1,46 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
Self-signed certifcates do not work at all, as well as the hardcoded keys. But nghttpx won't start without a key. So I just put a fake one here to make `service nghttpx start` work. Once the work on Let's Encrypt is done, this would be removed.